toolsets/nicgc/nicgc

```diff
@@ -1,60 +1,60 @@
-#!/bin/bash
+#!/bin/bash -e
```

Comment:
This script previously did not have the `-e` flag set in the prelude like most others do. I added it in this PR to address [Linter warning SC2164](https://www.shellcheck.net/wiki/SC2164) (performing a `cd` without checking if it succeeded/failed). Is there a reason to not include this flag for this utility script?